superset/utils/core.py

```diff
@@ -650,11 +671,13 @@ def get_email_address_list(address_string):
     if isinstance(address_string, basestring):
         if ',' in address_string:
             address_string = address_string.split(',')
+        elif '\n' in address_string:
+            address_string = address_string.split('\n')
         elif ';' in address_string:
             address_string = address_string.split(';')
         else:
             address_string = [address_string]
-    return address_string
+    return [x.strip() for x in address_string]
```

Comment:
`return [x.strip() for x in address_string if x.strip()]` would be event more defensive and prevent empty strings

Reply:
done